mewarpx/tests/test_capacitive_discharge.py

```diff
@@ -0,0 +1,87 @@
+"""Test full 1D diode run with diagnostics."""
+import pytest
+import numpy as np
+
+from mewarpx import mwxutil
+
+
+@pytest.mark.parametrize(
+    ("name"),
+    [
+        'Run2D',
+        'Run2D_RZ',
+        'Run3D'
+    ]
+)
+def test_run1D_alldiags(capsys, name):
+    basename = "Run"
+    use_rz = 'RZ' in name
+    dim = int(name.replace(basename, '')[0])
+
+    # Initialize and import only when we know dimension
+    mwxutil.init_libwarpx(ndim=dim, rz=use_rz)
+    from mewarpx import testing_util
+    from mewarpx.setups_store import diode_setup
+
+    # Include a random run number to allow parallel runs to not collide. Using
+    # python randint prevents collisions due to numpy rseed below
+    testing_util.initialize_testingdir(name)
+
+    # Initialize each run with consistent, randomly-chosen, rseed. Use a random
+    # seed instead for initial dataframe generation.
+    # np.random.seed()
+    np.random.seed(92160881)
```

Comment:
To avoid collisions, you could use the current unix time as the seed, that would save having to manually set the seed each run.

Reply:
This seed is for random numbers drawn in the physics (admittedly in the python side only), not for the directories! The latter explicitly don't use this seed. So having this seed constant is nice as calculations can be exactly the same.